teamcity: failed test: TestEntries

The following tests appear to have failed on master (testrace): TestEntries

You may want to check [for open issues](https://github.com/cockroachdb/cockroach/issues?q=is%3Aissue+is%3Aopen+TestEntries).

[#1362188](https://teamcity.cockroachdb.com/viewLog.html?buildId=1362188):


```
TestEntries
--- FAIL: testrace/TestEntries (0.170s)

------- Stdout: -------
I190626 23:26:10.753685 26593 util/stop/stopper.go:542  quiescing; tasks left:
1      [async] intent_resolver_ir_batcher
1      [async] intent_resolver_gc_batcher
I190626 23:26:10.753883 26593 util/stop/stopper.go:542  quiescing; tasks left:
1      [async] intent_resolver_ir_batcher
    replica_test.go:6687: 22: expected cache count 30, got 31
    replica_test.go:6696: 22: expected error requested entry at index is unavailable, got %!s(<nil>)




```

Please assign, take a look and update the issue accordingly.

--- a/pkg/storage/replica_test.go
+++ b/pkg/storage/replica_test.go
@@ -6567,9 +6567,13 @@ func TestQuotaPoolAccessOnDestroyedReplica(t *testing.T) {
 func TestEntries(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	tc := testContext{}
+	cfg := TestStoreConfig(nil)
+	// Disable ticks to avoid quiescence, which can result in empty
+	// entries being proposed and causing the test to flake.
+	cfg.RaftTickInterval = math.MaxInt32
 	stopper := stop.NewStopper()
 	defer stopper.Stop(context.TODO())
-	tc.Start(t, stopper)
+	tc.StartWithStoreConfig(t, stopper, cfg)
 	tc.repl.store.SetRaftLogQueueActive(false)
 
 	repl := tc.repl

